### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in hca end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7384]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the HCA tests, I noticed a few things worth a second look:

* Tests kept failing on the file upload step. I'm not sure this is an issue in the CI/CD pipeline, but it's worth a look to ensure all of our e2e tests and axe checks are running properly.
* We should open 1 accordion on the review page and then run the axe check.

I'd like the front-end team to review these tests and ensure we have the right level of coverage. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/hca/tests`
* `$ yarn test:e2e src/applications/hca/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks. Ensure the checks are not failing on the file upload and fix them if necessary.
- [x] Expand 1 accordion on the form review page, then run the axe check
- [x] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [x] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by AZSchneider** [comment]
Adding @luke-gcio to put on the Benefits-Memorials Team 1 backlog

**Comment by luke-gcio** [comment]
Got it!

**Comment by Mottie** [comment]
I've addressed everything except the upload issues. This appears to be an ongoing problem when using Nightwatch (e2e testing) with Selenium. This probably won't be an issue once we switch from using Nightwatch over to Cypress.

**Comment by Mottie** [comment]
This work is still in progress. The e2e tests for axe checks for HCA are failing because the axe-core library is severly out-of-date. I opened a [PR to update axe-core](https://github.com/department-of-veterans-affairs/vets-website/pull/13105), but that work is also having issues with non-accessibility related e2e tests failing. Once that has been resolved, this work can be updated and merged.

**Comment by Mottie** [comment]
My HCA PR may be obsolete soon... a PR for HCA e2e test using Cypress has been completed - https://github.com/department-of-veterans-affairs/vets-website/pull/13119 - and should be merged soon.
